*** piratebox/piratebox/conf/hostapd.conf	2017-03-19 15:55:17.000000000 +0000
--- /opt/piratebox/conf/hostapd.conf	2017-07-04 15:28:47.700000162 +0100
***************
*** 1,12 ****
  interface=wlan0
  driver=nl80211
! ssid=PirateBox - Share Freely
  
! channel=1
  #auth_algs=1
  
  # Remove # to enable n wifi mode
! #ieee80211n=1
  #wmm_enabled=1
  #ht_capab=[HT40+][SHORT-GI-40][DSSS_CCK-40]
  hw_mode=g
--- 1,12 ----
  interface=wlan0
  driver=nl80211
! ssid=PorangeBox
  
! channel=8
  #auth_algs=1
  
  # Remove # to enable n wifi mode
! ieee80211n=1
  #wmm_enabled=1
  #ht_capab=[HT40+][SHORT-GI-40][DSSS_CCK-40]
  hw_mode=g
