*** hostap.conf.org	2017-07-03 20:41:27.087592414 +0100
--- /opt/piratebox/conf/hostapd.conf	2017-07-03 20:09:12.311321034 +0100
***************
*** 6,12 ****
  #auth_algs=1
  
  # Remove # to enable n wifi mode
! #ieee80211n=1
  #wmm_enabled=1
  #ht_capab=[HT40+][SHORT-GI-40][DSSS_CCK-40]
  hw_mode=g
--- 6,12 ----
  #auth_algs=1
  
  # Remove # to enable n wifi mode
! ieee80211n=1
  #wmm_enabled=1
  #ht_capab=[HT40+][SHORT-GI-40][DSSS_CCK-40]
  hw_mode=g
